src/main/java/bio/terra/workspace/service/workspace/flight/CreateProjectStep.java

```diff
@@ -0,0 +1,151 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import static bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys.GOOGLE_PROJECT_ID;
+
+import bio.terra.cloudres.google.api.services.common.OperationCow;
+import bio.terra.cloudres.google.api.services.common.OperationUtils;
+import bio.terra.cloudres.google.cloudresourcemanager.CloudResourceManagerCow;
+import bio.terra.cloudres.google.serviceusage.ServiceUsageCow;
+import bio.terra.stairway.*;
+import bio.terra.stairway.exception.RetryException;
+import bio.terra.workspace.app.configuration.external.GoogleWorkspaceConfiguration;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.services.cloudresourcemanager.model.Project;
+import com.google.api.services.cloudresourcemanager.model.ResourceId;
+import com.google.api.services.serviceusage.v1.model.BatchEnableServicesRequest;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Optional;
+
+/**
+ * A {@link Step} for creating a Google Project for a workspace.
+ *
+ * <p>TODO(PF-156): Use RBS for project creation instead.
+ */
+public class CreateProjectStep implements Step {
+  @VisibleForTesting
+  static final ImmutableList<String> ENABLED_SERVICES =
+      ImmutableList.of("storage-api.googleapis.com");
```

Comment:
might this list - and the folder id - need to be passed in as arguments to the flight instead of hard-coded/in configuration?

Reply:
This project creation is all temporary since we're planning on migrating to RBS to buffer projects, so I'm inclined to make it a simple as possible in the short term.
I'm not sure if you're thinking that the folder and enabled services should be parameters to setting up a project, or exposed through the WM API. I'm going to assume the former.
Things like the folder id & enabled APIs (but medium term, which RBS pool to grab a project from) will be FlightMap inputParameters. It's not obvious how that will be reflected in the API yet. This relates to our "workspace profiles" to allow different project configuration for AoU vs CWB vs different Enterprise partners. (this is lightly touched in PF-158). I'm inclined to let us figure that out when we get further along and start simple for now. But yes, I expected there to be more parameters for how to set up a project.

Reply:
👍 , sounds like you've thought through it and I agree with the approach of keeping it simple for now and adding when necessary!